docs/README.md

```diff
@@ -1,29 +1,44 @@
 # Kubernetes Documentation
 
-**Note**
-This documentation is current for HEAD.
+## NOTE: This documentation is current for HEAD.
 
-Documentation for previous releases is available at
-http://releases.k8s.io/vX.Y.Z/docs.
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+If you are using a released version of Kubernetes, you almost certainly want
+the docs that go with that version.
+
+Documentation for specific releases is available at
+releases.k8s.io/v*X*.*Y*.*Z*/docs
+   * [releases.k8s.io/v0.21.0](http://releases.k8s.io/v0.21.0/docs)
```

Comment:
Not really.  Open for discussion, I guess.  It's more like "here are some
examples".
On Jul 7, 2015 7:17 PM, "Zach Loafman" <notifications@github.com> wrote:

> In docs/README.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10896#discussion_r34111169>
> :
>
> >
> > -Documentation for previous releases is available at
> > -http://releases.k8s.io/vX.Y.Z/docs.
> > +<!-- BEGIN STRIP_FOR_RELEASE -->
> > +
> > +If you are using a released version of Kubernetes, you almost certainly want
> > +the docs that go with that version.
> > +
> > +Documentation for specific releases is available at
> > +releases.k8s.io/v*X*.*Y*.*Z*/docs
> > +   * [releases.k8s.io/v0.21.0](http://releases.k8s.io/v0.21.0/docs) <http://releases.k8s.io/v0.21.0%5D(http://releases.k8s.io/v0.21.0/docs)>
>
> Is anything supposed to be keeping this list up to date?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10896/files#r34111169>
> .
>